src/platforms/python/common/configuration/integrations/sqlalchemy.mdx

```diff
@@ -25,23 +21,45 @@ pip install --upgrade 'sentry-sdk[sqlalchemy]'
 
 ## Configure
 
-Add `SqlalchemyIntegration()` to your `integrations` list:
-
 <SignInNote />
 
 ```python
 import sentry_sdk
-from sentry_sdk.integrations.sqlalchemy import SqlalchemyIntegration
 
 sentry_sdk.init(
     dsn="___PUBLIC_DSN___",
-    integrations=[
-        SqlalchemyIntegration(),
-    ],
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    traces_sample_rate=1.0,
 )
 ```
 
+The Redis integration is enabled automatically if you have the `sqlalchemy` package installed.
+
+## Verify
+
+```python
+from sqlalchemy import create_engine
+from sqlalchemy.sql import text
+
+def main():
+    sentry_sdk.init(...)  # same as above
+
+    engine = create_engine(DATABASE_URL, echo=True)
+    statement = text("SELECT 'Hello World'")
+
+    with engine.connect() as conn:
+        with sentry_sdk.start_transaction(name="testing_sentry"):
+            result = conn.execute(statement)
+
+main()
+```
+
+This will create a transaction called `testing_sentry` in the Performance section of Sentry and will create a span for `SELECT` statement.
+
+It takes a couple moments until the data shows up in Sentry.
```

Comment:
```suggestion
This will create a transaction called `testing_sentry` in the Performance section of [sentry.io](https://sentry.io) and will create a span for the `SELECT` statement.

It takes a couple of moments for the data to appear in [sentry.io](https://sentry.io).
```